packages/expo-blur/src/__tests__/__snapshots__/BlurView-test.ios.tsx.snap.ios

```diff
@@ -13,6 +13,7 @@ exports[`renders a native blur view 1`] = `
   testID="blur"
 >
   <ViewManagerAdapter_ExpoBlurView
+    blurReductionFactor={4}
```

Comment:
I've talked with @tsapete - we decided it will be better to rename that. Let's call it `androidBlurReductionFactor ` instead.